src/src/DataTypes/ESPEasyFileType.cpp

```diff
@@ -14,10 +18,27 @@ bool matchFileType(const String& filename, FileType::Enum filetype)
 
 bool isProtectedFileType(const String& filename)
 {
-  return matchFileType(filename, FileType::CONFIG_DAT) ||
-         matchFileType(filename, FileType::SECURITY_DAT) ||
-         matchFileType(filename, FileType::NOTIFICATION_DAT) ||
-         matchFileType(filename, FileType::PROVISIONING_DAT);
+  #if FEATURE_EXTENDED_CUSTOM_SETTINGS
+  bool isTaskSpecificConfig = false;
+  const String  fname       = filename.substring(filename.startsWith(F("/")) ? 1 : 0);
```

Comment:
Shouldn't we have an `unpatch_filename` function for this?

Reply:
We only need that ~2-3 times or so, so I don't think it's worth the overhead.